packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -103,25 +87,25 @@ The values for a configuration object are:
 
 The configuration information in a shipping app is usually static. You can initialize the configuration values directly in the code or read them from a file. During development you may want to provide a way to edit configuration values.
 
-<StackSnippet snippet="loadingaconfiguration" />
+<StackSnippet snippet="initializeflow" />
 
-### Start sign-in
+### Start the sign-in flow
 
-Start the sign-in flow by creating an SDK client, and then requesting the first step.
+Start the sign-in flow after initializing an `InteractionCodeFlow` object.
 
 <StackSnippet snippet="initializingsdksession" />
 
 ### Process the response
 
 The steps for processing a response are:
 
-1. Check for a successful sign-in flow.
+1. Check if the sign-in flow succeeded.
 1. Check for messages, such as an invalid password.
 1. Check for remediations.
 1. Process the remediations.
 1. Process the authenticators.
 
-After the user enters the required information, update the remediation and request the next step.
+After the user enters any required information, update the remediation and request the next step.
 
 <StackSnippet snippet="processresponse" />
 
```

Comment:
In line 114, do you not exchange the remediation flow for an ID token as well as an access token?